discovery-provider/ddl/migrations/0023_fix_broken_cover_art_sizes.sql

```diff
@@ -0,0 +1,18 @@
+WITH cover_art_cte AS (
+  SELECT track_id, owner_id, cover_art_sizes,
+  (
+    SELECT cover_art_sizes FROM tracks WHERE track_id = t1.track_id AND is_current = FALSE ORDER BY blocknumber DESC LIMIT 1
```

Comment:
might be worth having a check here to ensure this one's `cover_art_sizes` _doesn't_ have "file" or "blob" urls

Reply:
what does that add? we'll be in a wonky state either way right?

Reply:
it gives you the most recent non-blob, non-file CID? like if there's an edit of the existing track still with the broken CID, keep going back until you find the one with the real CID

Reply:
i think I worded it funny, but not so much "check" but a "where" clause

Reply:
i misunderstood what you meant. done!